src/modules/land_detector/FixedwingLandDetector.cpp

```diff
@@ -31,22 +31,23 @@
  *
  ****************************************************************************/
 
-/**
+/*
```

Comment:
It needs two starts to be considered a Doxygen comment. Please add the star back in.